charters/ExpressionMatrixService/charter.md

```diff
@@ -0,0 +1,90 @@
+
+# Expression Matrix Service
+
+## Description
+
+The expression matrix service enables researchers to easily access and analyze HCA expression matrix data.
+
+## Objective
+
+* Build, operate, and maintain a data access API that serves expression
+matrices from the HCA DCP Data Store with options to subset data based on bundle metadata, genes, and numerical filters.
+
+* Serve as a test bed for a re-usable metadata query service, created by
+the *Query Service* component.
+
+## In-scope
+
+* Expression Matrix Service interface supports queries based on:
+    * Bundle identifiers
+    * Metadata (*across bundles; potentially provided directly by Query Module*)
+    * Numerical filtering (*on gene expression values, and also QC metrics*)
+    * Compound metadata and numerical filtering
```

Comment:
I think I know what you mean by each of these points, but will the community? They could easily be expanded with a sentence or two each.